src/snowflake/connector/network.py

```diff
@@ -477,9 +478,6 @@ def request(
         else:
             accept_type = CONTENT_TYPE_APPLICATION_JSON
 
-        if timeout is None:
-            timeout = self._connection.network_timeout
-
```

Comment:
from the type hint timeout can still be none, so are we not specifying timeout anymore when it's a `ProxySupportAdapter`?

Reply:
I catch this in `fetch`:

```
timeout = (
    timeout if timeout is not None else self._connection.network_timeout
)
```